collectOutsideChanges, a synchronous function, awaits readFile, a syntax error in collect.js

--- a/src/turn/collect.js
+++ b/src/turn/collect.js
@@ -67,7 +67,7 @@
 
     if (getFileSize(copiedFile) > MAX_DIFF_BYTES) continue
 
-    addChange(collector, touchedFile, touchedFile, readFile(copiedFile))
+    addChange(collector, touchedFile, touchedFile, await readFile(copiedFile))
   }
 }
 
